Emoji/src/js/components/App/App.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+
+import Emoji from '../../../emojiList.json';
+
+import Form from '../Form/Form.jsx';
+import Select from '../Select/Select.jsx';
+import EmojiContainer from '../EmojiContainer/EmojiContainer.jsx';
+
+import './App.scss';
+
+const App = () => {
+
+    const [inputValue, setInputValue] = useState('');
+    const [numberOfEmoji, setNumberOfEmoji] = useState(10);
+
+    const handlerInputChange = (inputValue) => {
+        setInputValue(inputValue);
+    }
+
+    const handlerSelectChange = (numberOfEmoji) => {
+        setNumberOfEmoji(numberOfEmoji);
+    }
+
+    const filteredEmoji = Emoji.filter((emo) => {
+        return emo.title.toLowerCase().includes(inputValue.toLowerCase())||emo.keywords.toLowerCase().includes(inputValue.toLowerCase())
```

Comment:
операторы нужно разделять пробелами, а то код совсем не читается
```suggestion
        return emo.title.toLowerCase().includes(inputValue.toLowerCase()) || emo.keywords.toLowerCase().includes(inputValue.toLowerCase())
```